app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    components = command.split()
+    first_part, second_part, third_part = components
```

Comment:
The unpacking of `components` is correct, but ensure that the input string is validated to contain exactly three elements before unpacking. This is crucial to prevent runtime errors if the input does not meet the expected format. Consider adding a check before this line.